File: app/src/main/java/com/juniperphoton/myerlistandroid/api/response/GetOrderResponse.java
package com.juniperphoton.myerlistandroid.api.response;


import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

public class GetOrderResponse extends CommonResponse {

    @SerializedName("OrderList")
    private ArrayList<ListOrder> listOrder;

    public String getOrder() {
        if (listOrder != null && listOrder.size() > 0) {
            return listOrder.get(0).getOrder();
        }
        return null;
    }

    private class ListOrder {
        @SerializedName("list_order")
        private String order;

        public String getOrder() {
            return order;
        }
    }
}

File: app/src/main/java/com/juniperphoton/myerlist/api/response/GetOrderResponse.kt
package com.juniperphoton.myerlist.api.response

import com.google.gson.annotations.SerializedName

import java.util.ArrayList

class GetOrderResponse : CommonResponse() {
    @SerializedName("OrderList")
    private val listOrder: ArrayList<ListOrder>? = null

    val order: String?
        get() {
            if (listOrder != null && listOrder.size > 0) {
                return listOrder[0].order
            }
            return null
        }

    private class ListOrder {
        @SerializedName("list_order")
        val order: String? = null
    }
}